README.rst

```diff
@@ -4,10 +4,92 @@ magnum-capi-helm
 
 OpenStack Magnum driver using helm to create k8s clusters
 with Cluster API.
-The long stated aim of this driver is to merge
-in upstream Magnum:
+
+The driver uses capi-helm-charts to create the
+k8s resources needed to create a k8s cluster
+using Cluster API, including various useful
+add ons like a CNI and a monitoring stack.
+https://github.com/stackhpc/capi-helm-charts
+
+Note, the above helm charts are intended to be
+a way to share a reference method to create K8s
+on OpenStack. The charts are not expected or
+indented to be specific to Magnum. The hope is
+they can also be used by ArgoCD, Flux or Azimuth
+to create k8s clusters on OpenStack.
+
+Work on this driver started upstream aroun October 2021.
+After failing to get merged during Bobcat,
+we created this downstream repo as a stop gap to help
+those wanting to use this driver now.
 https://specs.openstack.org/openstack/magnum-specs/specs/bobcat/clusterapi-driver.html
 
+Installation and Dependencies
+=============================
+
+If you install this python package within your Magnum virtual env,
+it should be picked up by Magnum:::
+
+  git clone https://github.com/stackhpc/magnum-capi-helm.git
+  cd magnum-capi-helm
+  pip install -e .
+
+The driver requires access to a Cluster API management cluster.
+For more information, please see:
+https://cluster-api.sigs.k8s.io/user/quick-start
+
+To access the above Cluster API management cluster,
+you need to configure where the kubeconfig file
+lives:::
+
+  [capi_helm]
+  kubeconfig_file = /etc/magnum/kubeconfig
+
```

Comment:
```suggestion

If the environment you are running magnum in doesn't have access to the internal openstack API, you'll also need to add:::

  [nova_client]
  endpoint_type = publicURL

```

Reply:
are, interesting... that is a good point. Actually, I think it probably should always be the publicURL there for magnum, given how we re-use that config. We should link to this:
https://github.com/stackhpc/magnum-capi-helm/blob/32139954f64a0f6af94eb74c202f41a8cc8c7f07/magnum_capi_helm/common/app_creds.py#L65C80-L65C80